tardis/io/model_reader.py

```diff
@@ -533,3 +536,394 @@ def parse_csv_abundances(csvy_data):
+
+    Returns
+    -------
+    runner_dict : dict
+    integrator_settings : dict
+    v_packet_settings : dict
+    virtual_spectrum_spawn_range : dict
+    """
+
+    runner_dict = {
+        "Edotlu_estimator": runner.Edotlu_estimator,
+        "bf_heating_estimator": runner.bf_heating_estimator,
+        "disable_electron_scattering": runner.disable_electron_scattering,
+        "enable_full_relativity": runner.enable_full_relativity,
+        "enable_reflective_inner_boundary": runner.enable_reflective_inner_boundary,
+        "inner_boundary_albedo": runner.inner_boundary_albedo,
+        "input_energy": runner.input_energy,
+        "input_mu": runner.input_mu,
+        "input_nu": runner.input_nu,
+        "input_r_cgs": runner.input_r,
+        "j_blue_estimator": runner.j_blue_estimator,
+        "j_estimator": runner.j_estimator,
+        "last_interaction_in_nu": runner.last_interaction_in_nu,
+        "last_interaction_type": runner.last_interaction_type,
+        "last_line_interaction_in_id": runner.last_line_interaction_in_id,
+        "last_line_interaction_out_id": runner.last_line_interaction_out_id,
+        "last_line_interaction_shell_id": runner.last_line_interaction_shell_id,
+        "line_interaction_type": runner.line_interaction_type,
+        "nu_bar_estimator": runner.nu_bar_estimator,
+        "photo_ion_estimator": runner.photo_ion_estimator,
+        "photo_ion_estimator_statistics": runner.photo_ion_estimator_statistics,
+        "r_inner": runner.r_inner_cgs,
+        "r_outer": runner.r_outer_cgs,
+        "seed": runner.seed,
+        "spectrum_frequency_cgs": runner.spectrum_frequency,
+        "spectrum_method": runner.spectrum_method,
+        "stim_recomb_cooling_estimator": runner.stim_recomb_cooling_estimator,
+        "stim_recomb_estimator": runner.stim_recomb_estimator,
+        "time_of_simulation_cgs": runner.time_of_simulation,
+        "use_gpu": runner.use_gpu,
+        "v_inner": runner.v_inner_cgs,
+        "v_outer": runner.v_outer_cgs,
+        "virt_logging": runner.virt_logging,
+        "virt_packet_energies": runner.virt_packet_energies,
+        "virt_packet_initial_mus": runner.virt_packet_initial_mus,
+        "virt_packet_initial_rs": runner.virt_packet_initial_rs,
+        "virt_packet_last_interaction_in_nu": runner.virt_packet_last_interaction_in_nu,
+        "virt_packet_last_interaction_type": runner.virt_packet_last_interaction_type,
+        "virt_packet_last_line_interaction_in_id": runner.virt_packet_last_line_interaction_in_id,
+        "virt_packet_last_line_interaction_out_id": runner.virt_packet_last_line_interaction_out_id,
+        "virt_packet_nus": runner.virt_packet_nus,
+        "volume_cgs": runner.volume,
+    }
+
+    try:
+        runner_dict["single_packet_seed"] = runner.single_packet_seed
+    except AttributeError:
+        runner_dict["single_packet_seed"] = None
+
+    for key, value in runner_dict.items():
+        if key.endswith("_cgs"):
```

Comment:
I'm a little confused here, I was under the impression that variable names ending in `_cgs` were unitless arrays and the `_cgs` implied that the units of the array was cgs.  Are there any attributes that trigger this conditional?

Additionally, it might be useful to just ignore all units and just save everything as unitless arrays in cgs units.  The reader will have to know which units correspond to the arrays when read in but this can be hard coded with something like a dictionary mapping.  I think avoiding saving the units as well would make the directory structure a bit simpler and more in line with the way that the simulation hdf is currently output.